mt-kit/core/svelte/src/lib/svelte/components/map/Map.svelte

```diff
@@ -19,6 +19,7 @@
   export let startCoordinates = DEFAULT_START_COORDINATES
   export let startZoom = ZOOM_NORWAY
   export let goToMapSkipLinkText = 'Gå til kart'
+  export let rotation = false
```

Comment:
```suggestion
  export let enableRotation = false
```
Maybe just expose the same variable name?